packages/core/src/components/carousel/carousel.tsx

```diff
@@ -64,15 +73,34 @@ export class AtomCarousel {
     this.swiperEl.swiper?.on('navigationNext', () => {
       this.atomClickNext.emit()
     })
+
+    let params: { pagination: PaginationOptions } = this.pagination && {
+      pagination: {
+        type:
+          this.paginationType === 'thumbnails' ? 'custom' : this.paginationType,
+        clickable: this.paginationClickable,
+      },
+    }
+
+    if (this.paginationType === 'thumbnails') {
+      const urls = !this.thumbnailImages ? [] : this.thumbnailImages.split(',')
```

Comment:
**suggestion:** you could avoid logical not here
```suggestion
      const urls = this.thumbnailImages ? this.thumbnailImages.split(',') : []
```

Reply:
done at [0074bf1](https://github.com/juntossomosmais/atomium/pull/290/commits/0074bf15b4fd27067c157150c239c9af6cacd256)